README.md

```diff
@@ -1,57 +1,73 @@
-# Project Name
+# React Single-page Application built with MSAL React and Microsoft identity platform
 
-(short, 1-3 sentenced, description of the project)
+This sample demonstrates how to use [MSAL React](https://www.npmjs.com/package/@azure/msal-react) to login, logout, conditionally render components to authenticated users, and acquire an access token for a protected resource such as Microsoft Graph.
 
 ## Features
```

Comment:
possibly redundant given the summary above?

Reply:
The other samples have a blurb like this, I think it's fine to give a quick summary, but I'll give you final say.

Reply:
just a nitpick, let's keep it then!